iaso/api/instances.py

```diff
@@ -237,8 +239,10 @@ def get_row(instance, **kwargs):
                 idict.get("id"),
                 file_content.get("_version") if file_content else None,
                 idict.get("export_id"),
-                idict.get("latitude"),
-                idict.get("longitude"),
+                org_unit.get("latitude") if not idict.get("latitude") else idict.get("latitude"),
+                org_unit.get("longitude") if not idict.get("longitude") else idict.get("longitude"),
+                org_unit.get("altitude") if not idict.get("altitude") else idict.get("altitude"),
```

Comment:
err that's quite the change in behavior, no? are we sure we want this and that the users are properly informed? @madewulf

Reply:
Move into frontend